source/community/reactnative/src/components/containers/container-wrapper.js

```diff
@@ -64,13 +64,19 @@ export class ContainerWrapper extends React.PureComponent {
             };
         }
         // padding
-		const padding = this.props.configManager.hostConfig.getEffectiveSpacing(Enums.Spacing.Padding);
+        const padding = this.props.configManager.hostConfig.getEffectiveSpacing(
+            Enums.Spacing.Padding,
+        );
         return (
             <React.Fragment>
                 <BackgroundImage
                     backgroundImage={this.payload.backgroundImage}
                 />
-                <View style={{paddingTop: padding, paddingBottom: padding}}>
+                <View
```

Comment:
Check if padding needs to be added in all the scenarios(or only backgroundimage)

Reply:
@pavankumarj Rest of the changes looks good to me. Please verify backgraoundImage scenario alone. Once confirmed, I will approve the PR

Reply:
I have added a check for backgroundImage support and that will take care of additional padding and alignment requirements